### ISSUE TITLE
Alignment types and documentation (Create)SelectInput

### ISSUE THREAD

**Issue by planetcrypton** [commercetools/ui-kit#2204]
**Describe the bug**
Documentation and types of CreateSelectInput and SelectInput seem not aligned:
The prop `isRequired` is not available according to the type of the props, but it is present in the documentation.
The prop `menuPortalZIndex` is not an optional prop according to its type, but is according to the documentation.

**To Reproduce**
Compare the props in this link:
https://uikit.commercetools.com/?path=/story/components-fields-selectfields--creatableselectfield
to this link
https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/creatable-select-input/src/creatable-select-input.tsx

**Expected behavior**
I expected the prop 'descriptions' of types and documentation to be aligned.

**Additional context**
Basically I'd just like to know what is intended and what I should be going after 😀: 
- Can / can't I use the prop `isRequired`?
- Should / Shouldn't I set the prop `menuPortalZIndex`?

Thanks for helping!
cheers
Sjoerd
